pages/linux/cpufreq-info.md

```diff
@@ -0,0 +1,35 @@
+# cpufreq-info
+
+> A tool to show CPU frequency information.
+
+- Show CPU frequency kernel information for all CPUs:
```

Comment:
In the description, it is mentioned "CPU frequency information", but here you write "CPU frequency kernel information". Why is the word "kernel" added ?

Reply:
It's a mistake. I;ve fixed it.